src/lib/components/Modals/LoginModal.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { useForm, DeepMap, FieldError } from 'react-hook-form';
+import {
+  Button,
+  TextField,
+  Grid,
+  Typography,
+  Link,
+  FormControlLabel,
+  Checkbox,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+} from '@material-ui/core';
+import { RegistrationModal } from './RegistrationModal';
+
+export interface IInputs {
+  email: string;
+  password: string;
+}
+
+export const LoginModal: React.FC = () => {
+  const [loginOpen, setLoginOpen] = React.useState(false);
+  const [registrationOpen, setRegistrationOpen] = React.useState(false);
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, errors } = useForm<IInputs>();
+
+  const handleLoginOpen = () => {
+    setLoginOpen(true);
+  };
+
+  const handleLoginClose = () => {
+    setLoginOpen(false);
+  };
+
+  const handleRegistrationOpen = (
+    event: React.MouseEvent<HTMLSpanElement, MouseEvent>,
+  ) => {
+    event.preventDefault();
+    setRegistrationOpen(true);
+  };
+
+  const handleRegistrationClose = () => {
+    setRegistrationOpen(false);
+    setLoginOpen(false);
+  };
+
+  const onSubmit = (data) => {
+    console.log(data);
+    console.log(errors);
+  };
+
+  const showErrorMessage = (
+    errorsObj: DeepMap<IInputs, FieldError>,
+    inputName: string,
+  ): JSX.Element => {
+    return (
+      errorsObj[inputName] && (
+        // eslint-disable-next-line
+        <span style={{ color: 'red' }}>{errorsObj[inputName].message}</span>
+      )
+    );
+  };
+
+  return (
+    <>
+      <Button variant="outlined" color="primary" onClick={handleLoginOpen}>
+        Log in/Register
+      </Button>
+
+      <Dialog
+        open={loginOpen}
+        onClose={handleLoginClose}
+        aria-labelledby="form-dialog-title"
+      >
+        <DialogTitle id="form-dialog-title">
+          Введіть Ваші email та пароль
+          {/* {response.error && (
+                  <span style={{ color: 'red' }}>Неправильний email або пароль</span>
+                )} */}
```

Comment:
remove

Reply:
This part will be uncommented when there will be connect with BE: it's a message in case of bad response from server (email and/or password turn out to be wrong).